### ISSUE TITLE
GoogleFnet fails with stack dump in E2E performance CI

### ISSUE THREAD

**Issue by sommerlukas** [intel/intel-xpu-backend-for-triton#1731]
In CI runs https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/10108590708 and https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/10108584426, the `GoogleFnet` model fails in various combinations of training/inference and datatype. 

The log shows that the benchmark causes a stack dump in Triton. More details can be found in the respective logs of the CI runs.

The failing configurations are:
```
67   xpu  huggingface                     GoogleFnet   training  bfloat16         0.0         0.0
159  xpu  huggingface                     GoogleFnet  inference  amp_bf16         0.0         0.0
205  xpu  huggingface                     GoogleFnet  inference  bfloat16         0.0         0.0
251  xpu  huggingface                     GoogleFnet  inference  amp_fp16         0.0         0.0
343  xpu  huggingface                     GoogleFnet   training   float16         0.0         0.0
435  xpu  huggingface                     GoogleFnet   training  amp_fp16         0.0         0.0
297  xpu  huggingface                     GoogleFnet   training  amp_bf16         0.0         0.0
21   xpu  huggingface                     GoogleFnet  inference   float16         0.0         0.0
113  xpu  huggingface                     GoogleFnet  inference   float32         0.0         0.0
389  xpu  huggingface                     GoogleFnet   training   float32         0.0         0.0
```

Additionally, a warning is printed before execution of the model: `GoogleFnet supports float32 only`. Therefore, it might make sense to focus the investigation on the failing configurations with `float32` datatype.

**Comment by vlad-penkin** [comment]
Switch to Upstream PyTorch fixes this issue

**Comment by pbchekin** [comment]
Fixed by #1952.
Successful run: https://github.com/intel/intel-xpu-backend-for-triton/actions/runs/10477366151.

**Comment by pbchekin** [comment]
No longer reproducible with PyTorch 2.5 (no IPEX). This is the default mode for E2E accuracy and performance tests (implemented in #1952).
